### ISSUE TITLE
[FEATURE REQUEST] Ability to omit [Config|Platform] in lines of build report.

### ISSUE THREAD

**Issue by gamagan** [mike-ward/VSColorOutput64#14]
Is it even possible to perform a build where different projects build with different debug/release/config settings at the same time? I'm not aware of that being possible.

For me, the addition of [Config|Platform] to every line of the 'Projects build report' is visually noisy.
Can you please update that output to only add that to the header?

Instead of this:
```
Projects build report:
  Status    | Project [Config|platform]
 -----------|---------------------------------------------------------------------------------------------------
  Succeeded | ZERO_CHECK [Debug|x64]
  Succeeded | Library [Debug|x64]
  Succeeded | Core [Debug|x64]
```

Have this:
```
Projects build report:
  Status    | Project [Debug|x64]
 -----------|---------------------------------------------------------------------------------------------------
  Succeeded | ZERO_CHECK 
  Succeeded | Library 
  Succeeded | Core 
```

Or, at least, have an option to select that output format.

Regards

**Comment by mike-ward** [comment]
You're right. I'll make the recommended changes.

**Comment by mike-ward** [comment]
Available in V2023.2

**Comment by Hilarius86** [comment]
You can do "Build > Batch Build" to build a complete mix of configs and platforms.
I just updated to VS2022 (with ColorOutput) and liked the feature that is now missing.

I also have lots of projects that I build and the header is rarely visible. So the information in each line was helpful.
Maybe you like to bring the feature back as an option?
